### ISSUE TITLE
[Feature Request]: Cross-Device Syncing

### ISSUE THREAD

**Issue by doug-stewart** [jordanbaird/Ice#95]
### Description

I have a personal Mac and a work Mac. A great feature would be settings syncing via iCloud.

**Comment by ceagnew** [comment]
Yes, please add iCloud syncing or some type of syncing.  Thanks for the consideration!

**Comment by hcbille** [comment]
Same usecase and feature request as the above mentioned. 
I've got multiple mac devices and I try to keep them set up the same, and I'd love if the Ice settings were syncable via iCloud or also just importable from a settings file I could keep in my config repo.

**Comment by KristijanZic** [comment]
Please try to make these profiles portable via configuration files somewhere in ./config/iced/iced.json/toml 
Whatever changes are configured in the app should be reflected in config file and whatever changes are made in the config file should be reflected in the app.
